tests/e2e/tidbcluster/serial.go

```diff
@@ -603,4 +462,68 @@ var _ = ginkgo.Describe("[tidb-operator][Serial]", func() {
 		})
 	})
 
+	// tidb-operator with AdvancedStatefulSet feature enabled and pod admission webhook enabled
+	ginkgo.Context("[Feature: AdvancedStatefulsetPodAdmissionWebhook]", func() {
+		var ocfg *tests.OperatorConfig
+		var oa tests.OperatorActions
+		var genericCli client.Client
+
+		ginkgo.BeforeEach(func() {
+			ocfg = &tests.OperatorConfig{
+				Namespace:      "pingcap",
+				ReleaseName:    "operator",
+				Image:          cfg.OperatorImage,
+				Tag:            cfg.OperatorTag,
+				SchedulerImage: "k8s.gcr.io/kube-scheduler",
+				Features: []string{
+					"StableScheduling=true",
+					"AdvancedStatefulSet=true",
+				},
+				LogLevel:          "4",
+				ImagePullPolicy:   v1.PullIfNotPresent,
+				TestMode:          true,
+				WebhookEnabled:    true,
+				PodWebhookEnabled: true,
```

Comment:
is it possible to enable `PodWebhookEnabled` in default operator?
https://github.com/pingcap/tidb-operator/blob/master/tests/e2e/config/config.go#L105

Reply:
If we enable `PodWebhookEnabled` in default, I think we will not cover the logic for previous controller.